packages/sandbox/src/app/sandbox/docs/navigation-bar-trail.doc.tsx

```diff
@@ -0,0 +1,69 @@
+import { NavigationBarTrail, IconsName } from '@getluko/streamline';
+
+import { DocList } from '../components/DocList';
+
+const left = {
+  accessibilityLabel: 'Go back',
+  iconName: 'ChevronLeft' as IconsName,
+  onPress: async () => {
+    console.log('pressed');
+  },
+};
+
+const right = {
```

Comment:
`const right:  ButtonIconProps = {` to prevent having the `as IconsName` and infer type